SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,21 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # GDK for Unreal directory structure
-The table below contains the contents of the built GDK for Unreal after running `BuildGDK.bat`.
+The table below contains the contents of the Unreal GDk repo folder after running `Setup.bat`.
 
 | Directory | Purpose
 |-----------|---------
-| `Binaries/ThirdParty/Improbable/` | (Not tracked in git) This folder contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
-| `build/` | (Not tracked in git) Intermediate folder used for build steps while running `BuildGDK.bat`.
-| `linting/` | Contains the linting scripts for checking the GDK for Unreal.
-| `docs` | Contains the documentation for the GDK for Unreal.
-| `fastbuild` | Contains files related to installing FASTBuild. **Note** these currently only work for Improbable engineers.
-| `packages/` | (Not tracked in git) Contains the dependencies for building the Unreal SDK codegen.
-| `Plugins/SpatialGDK/SpatialGDKEditorToolbar/` | The [SpatialOS GDK for Unreal toolbar]({{urlRoot}}/content/toolbar.md) within the Unreal Editor, from which you can take snapshots, generate interop code, start and stop deployments, and access the SpatialOS Inspector.
-| `schema` | Contains the schema files required for the GDK to interact with SpatialOS.
-| `Scripts/` | Contains a set of helper scripts used to build a GDK worker.
-| `Source/Programs/` | Contains the Unreal SDK code generator.
-| `Source/SpatialGDK/Legacy/` | Contains the source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS).
-| `Source/SpatialGDK/Legacy/Deprecated/` | Contains the forked source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS) that the GDK does not depend on. This is present purely for backwards compatibility.
-| `Source/SpatialGDK/Public/WorkerSdk/` | (Not tracked in git) Contains the [C++ worker SDK](https://docs.improbable.io/reference/latest/cppsdk/introduction) headers. You install these when you run `BuildGDK.bat`
-| `Source/SpatialGDK/Public` | Public source code of the GDK for Unreal module.
-| `Source/SpatialGDK/Private` | Private source code of the GDK for Unreal module.
+| `SpatialGDK/Binaries/ThirdParty/Improbable/` | (Not tracked in git) This folder contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
+| `SpatialGDK/Build/core_sdk/` | (Not tracked in git) Contains the SpatialOS C API dependencies.
```

Comment:
Is it worth saying that the SpatialOS C API is the foundation for the GDK? 

"SpatialOS C API dependencies" doesn't really mean anything to anyone unless they already know that the C API is the foundation of the SpatialOS GDKs and SDKs.

Reply:
Please also hyperlink C API to https://docs.improbable.io/reference/13.2/capi/introduction

Reply:
I ummed and aahed about this  - I am not sure whether this is useful to users here?! Are they likely to actually need to use it?
If it is useful, it needs to be https://docs.improbable.io/reference/latest/capi/introduction

Reply:
I ummed and aahed about this - is this useful to users? Are they going to use the C API? Do we want to go there without explaining how you might use it?
If we do want to link to there, then use 'latest'.
https://docs.improbable.io/reference/latest/capi/introduction